test/extensions/transport_sockets/tls/ssl_socket_test.cc

```diff
@@ -2438,6 +2439,10 @@ void testTicketSessionResumption(const std::string& server_ctx_yaml1,
             dynamic_cast<const SslSocketInfo*>(client_connection->ssl().get());
         ssl_session = SSL_get1_session(ssl_socket->rawSslForTest());
         EXPECT_TRUE(SSL_SESSION_is_resumable(ssl_session));
+        if (expect_reuse && expected_lifetime_hint) {
```

Comment:
`expect_reuse` shouldn't be needed here, since session timeout is also available in the original session, not only in the resumed one.